src/cli/arch/cli.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+FILE=/usr/share/pam-configs/Facerec
+SUDO=/etc/pam.d/sudo
+TEXT='auth      sufficient   pam_python.so /lib/Auth/Facerec/pam.py'
+
+function facerec(){
+if [ "$1" = "new" ]; then
+    sudo python3 /usr/lib/Auth/Facerec/add_new.py
+
+
+elif [ "$1" = "enable" ]; then
+    if grep -q $TEXT $SUDO;then
+      echo "alred"
+    else
+      sudo echo $TEXT >> $SUDO
+      echo "done"
+    fi
+    echo "Enabling facerec... Done"
+
+
+elif [ "$1" = "disable" ]; then
+    if grep -q $TEXT $SUDO;then
+      sed -i $TEXT $SUDO
+    else
+      echo "NotE"
```

Comment:
Looks like a typo!?

Reply:
I have removed the block, as it was not functing as expected during installation, instead user has to manually ass the following line `auth      sufficient   pam_python.so /lib/Auth/Facerec/pam_ptn.py` to `/etc/pam.d/sudo` to enable and diable the authentication package